packages/record-data/addon/-private/relationships/state/relationship.ts

```diff
@@ -620,7 +610,7 @@ export default class Relationship {
    `push` use `updateMeta`, `updateData` and `updateLink` to update the state
    of the relationship.
    */
-  push(payload: JsonApiRelationship, initial?: boolean) {
+  push(payload: JsonApiRelationship) {
```

Comment:
Same here - never passed as an argument.

This seems to be accomplished with `addRecordData` && `addCanonicalRecordData`

Reply:
https://github.com/emberjs/data/blob/master/packages/record-data/addon/-private/record-data.ts#L223

A specific use case to show `initial` is not a used argument for `push`